### PR TITLE
Add some docs, make some vars static, improved and fixed setMultiColo…

### PR DESCRIPTION
**Pull request by lBeJIuk**
Hey, I also had a problem with the wrong productId. so first I added an array where you can store known productId's. 
I also put a few control sequences and mappings into static variables.
And at the end, I did a little remaking of setMultiColorLed.
Now if you do not pass values for a certain button, it will be considered zero.
I also deleted `const ledMatrixFlattened = ledMatrix.flat();` since I didn't understand why it was there. Maybe it's an error?

### CODE REVIEW

**Comment by manualmanul** [comment]
Thank you for your contribution! You've made a lot of good changes, I'll review them later and merge if everything is okay.

> I also deleted const `ledMatrixFlattened = ledMatrix.flat();` since I didn't understand why it was there. Maybe it's an error?

I've made the function accept a matrix of keys where each row of keys was in a separate array, so by calling .flat() you'd get rid of those rows. My thinking was that this would make it easier to understand how to use this function, and internally I'd disregard this structure and just use matrix_id from the AnnePro2Layout object.

Do you think it would be better to use a 2D matrix like before or just an array of RGB values for each key (like in your MR)?

**Comment by lBeJIuk** [comment]
Yes, now I understand the intentions.
It's hard to say what would be more convenient to use. But first, I'd keep your version.

**Comment by lBeJIuk** [comment]
Oh, I just looked at the Array.prototype.flat support. According to this resource https://node.green/, version 10 does not support flat. So it's necessary to replace the description in the Readme, or to use the alternative to flat.

**Comment by manualmanul** [comment]
You're right, I'll update the README to mention Node v12 as the minimum version

File: index.js [hunk 1]
```diff
@@ -138,31 +66,150 @@ class LEDController {
...
+setInterval(() => {
+  if (counter % 2 === 0) {
+    lc.setMultiColorLed([[255,0,0],[0,255,0],[0,0,255]]);
+    // lc.setSingleColorLed([0, 255, 255]);
+  } else {
+    lc.setSingleColorLed([0, 0, 0]);
+  }
+  counter++;
+}, 1000);
+
```
Comment (manualmanul) [inline index.js:214]: I think you've forgotten to remove some of your test code

Comment (lBeJIuk) [reply index.js]: Yeah, you're right. I forgot to delete it. Fixed it.

**Pull request merged**